templates/twitter/grafbase/schema.graphql

```diff
@@ -0,0 +1,31 @@
+type User @model {
+  username: String! @unique
+  email: Email! @unique
+  url: URL
+  avatar: String
```

Comment:
Use URL scalar here?

Reply:
I opted not to use `URL` here since the image could be relative to the application using this schema.

Reply:
I would stick to URL since that's what the Twitter API uses.